### PR TITLE
[Rename Classes] LPS-166962 Rename classes in modules layout-api、layout-impl、headless-delivery-impl、item-selector-taglib、frontend-js-bundle-config-extender

### PR DESCRIPTION
**Pull request by W2jx**

### CODE REVIEW

**Comment by Kvale1733** [comment]
@W2jx No regressions found.

File: modules/apps/layout/layout-taglib/src/main/java/com/liferay/layout/taglib/internal/servlet/ServletContextUtil.java [hunk 1]
```diff
@@ -106,8 +106,8 @@ public static InfoListRendererTracker getInfoListRendererTracker() {
 		return _layoutDisplayPageProviderTracker;
 	}
 
-	public static LayoutListRetrieverTracker getLayoutListRetrieverTracker() {
-		return _layoutListRetrieverTracker;
+	public static LayoutListRetrieverRegistry getLayoutListRetrieverTracker() {
```
Comment (dantewang) [inline modules/apps/layout/layout-taglib/src/main/java/com/liferay/layout/taglib/internal/servlet/ServletContextUtil.java]: @W2jx Please also rename methods that returns instance of the classes you renamed.

**Review by dantewang — CHANGES_REQUESTED** [review]

**Comment by dantewang** [comment]
@W2jx Never mind, I didn't see you rename commit next.

**Comment by dantewang** [comment]
PR sent to https://github.com/brianchandotcom/liferay-portal/pull/125764

@W2jx See https://github.com/brianchandotcom/liferay-portal/pull/125764/commits/d0f8a9f5bf58b135d8f888937ae76a04416db3c0, it's a major version bump.

**Pull request closed without merging**